2018-11/index.md

```diff
@@ -4,69 +4,69 @@ title: BeagleBoard.org Enables Success
+   image: https://beagleboard.org/blog/wp-content/uploads/2018/10/Poursteady-at-show.png
+ - title: 10 Year Anniversary Classic Project - Three Story Tall Clock
+   link: https://beagleboard.org/p/bigjosh/three-story-tall-clock-final-entry-5ab263
+   body: An iconic building-sized digital Rudolph de Harak clock using 129,600 independent RGB LEDs controlled by 72 BeagleBone® Green boards provides an amazing artistic and engineering feat.
+   image: http://beagleboard.org/static/newsletter/images/clock.png
+ - title: Educator Highlight - Derek Molloy, Embedded Linux YouTube Rock-Star and Author
+   link: https://beagleboard.org/blog/2018-10-21-derek-molloy
+   body: Dr Molloy not only teaches students at his local university, he also shares his teaching materials with over one million YouTube viewers.  In this interview, he reveals why and how BeagleBone helps students and instructors succeed.
+   image: https://beagleboard.org/blog/wp-content/uploads/2018/10/Screen-Shot-2018-10-19-at-8.33.03-AM.png
+ - title: Empowering Innovative Education & Celebrating Community (Maker Faire NYC)
+   link: https://beagleboard.org/blog/2018-10-18-empowering-innovative-education-celebrating-community-maker-faire-nyc
+   body: World Maker Faire is an inspiring location for meeting with the BeagleBoard.org® community.  Check out the demos, talks and celebrations we shared.  Get started with PocketBeagle® using the educational workshop materials.
+   image: https://beagleboard.org/blog/wp-content/uploads/2018/10/01_MFNYboothcrew280_1.jpg
 topics:
- - title: Adding Headers to Your PocketBeagle®?
-   link: https://groups.google.com/forum/embed/?&parenturl=https%3A%2F%2Fbeagleboard.org%2Fpocket&showsearch=true&showpopout=true&showtabs=true&place=forum/beagleboard#!category-topic/beagleboard/pocketbeagle/Uxx_9ce2YHM
-   body: Check out the options discussed in this thread to help with the choices.
- - title: Back to School with Embedded Linux on BeagleBone®
-   link: https://groups.google.com/forum/embed/?place=forum/beagleboard&showsearch=true&showpopout=true&showtabs=false&hideforumtitle=true&parenturl=https%3A%2F%2Fbeagleboard.org%2Fdiscuss%2F%3Fplace%3Dtopic%252Fbeagleboard%252FoMCYbQgzjQo%252Fdiscussion#!category-topic/beagleboard/beaglebone-black/XJeKUBAzivI
-   body: Check out with Prof Yoder’s course with materials posted in this thread.
- - title: Qt5 "hello world" how-to on BeagleBone Debian
-   link: https://groups.google.com/forum/embed/?place=forum/beaglebone&showsearch=true&showpopout=true&showtabs=false&hideforumtitle=true&parenturl=https%3A%2F%2Fbeagleboard.org%2Fdiscuss%2F%3Fplace%3Dtopic%2Fbeagleboard%2FoMCYbQgzjQo%2Fdiscussion#!category-topic/beagleboard/software/bGuAcHm0atc
-   body: Need to use Qt on an embedded system – check this post for latest getting started
+ - title: Fresh Debian 9.5 Images
+   link: https://beagleboard.org/blog/2018-10-10-fresh-debian-9-5-images
```

Comment:
```suggestion
   link: https://bbb.io/@2019
```

You don't have to use the shortlinks, but they are handy and I'd like to promote them. You can grab the shortlinks off of the wp-admin posts page by clicking "Get Shortlink".

Reply:
okay will do next time